Update filtering.md per style guide (#512)
bigcommerce/dev-docs: docs/api-docs/getting-started/filtering.md

To filter collections down to a particular set of items, you can add filters to your request as URL query parameters.

> The maximum number of products returned is 250. A script is needed to loop through each page to view more than that.

Reviewer: ```suggestion
The maximum number of products returned is 250. Viewing more products requires creating a script that can loop through each subsequent page.

```

## Includes

Some endpoints allow `?include` parameter to include sub-resources and other information in the primary GET response for a parent object.

These endpoints allow you to save API calls by getting more information in a response. However, it may slow down your response.

For example, you can include a product's variants and images with the product response: `/v3/catalog/products?include=variants,images`

> The availability of the `?include` parameter varies by an endpoint. Refer to the `GET` method for an endpoint to see a list of available fields.

Reviewer: ```suggestion
The availability of the `?include` parameter varies by endpoint. Refer to the `GET` method for an endpoint to see a list of available fields.

```

Some endpoints allow an `?include` parameter to include sub-resources and other information in the primary GET response for a parent object.

These endpoints allow you to save API calls by getting more information in a response. However, it may slow down your response.

For example, you can include a product's variants and images with the product response: `/v3/catalog/products?include=variants,images`

> The availability of the `?include` parameter varies by an endpoint. Refer to the GET method for an endpoint to see a list of available fields.

Reviewer 2: remove  `an`

Reviewer 2: add in-line code fence around HTTP verbs, example:

```markdown
`GET`
```

### Including and Excluding Fields

Some endpoints support both `?include_fields` and `?exclude_fields`. 

`include_fields` will return ONLY the specified fields in the response. `exclude_fields` will omit the specified fields from the response.

> You can specify any field that is available on the object. Excluding fields you don't care about (unusually large fields like descriptions) can speed up your API request response time.

Reviewer 2: Remove `unusually` and add back `especially`.

**Example Response Get Product Name and Price**  

> Here, product name and price information are in the endpoint:

Reviewer 2: this change isn't necessary.

Author: I would like to keep this change because I would like to remove passive voice as much as possible.
